README.md

```diff
@@ -113,15 +113,28 @@ If you are an Enterprise user with first-party hosting access, you will need to
 You can then enable it in your code:
 
 ``` swift
-public enum Endpoint {
-    case default, alternate
-}
+let hcaptcha = try? HCaptcha(
+    ...
+    endpointApi: URL("https://custom.api.endpoint")!,
+    endpoint: URL("https://custom.endpoint")!,
+    ...
+)
 
-let hcaptcha = try? HCaptcha(endpoint: .alternate) // Defaults to `default` when unset
+...
 ```
 
-after setting your URI in HCaptcha/Classes/HCaptcha.swift: `altjsurl` variable.
+#### More params for Enterprise
+
+More params like:
+ - `sentry`
+ - `rqdata`
+ - `reportapi`
+ - `assethost`
+ - `imghost`
+
+Can be passed via `HCaptcha(...)`
 
+Checout documentation in docs and https://docs.hcaptcha.com/api for more details
```

Comment:
@e271828- This link looks relevant, but I would like to doublecheck if it make sense here